Source/gg2/Objects/InGameElements/Sentry.events/User Event 13.xml

```diff
@@ -27,6 +27,13 @@ if(global.updateType == QUICK_UPDATE or global.updateType == FULL_UPDATE) {
     tbyte = read_ubyte(global.deserializeBuffer);
     built = (tbyte &amp; $80) != false;
     hp = tbyte &amp; $7F;
+    if(!built and global.updateType == FULL_UPDATE)
+    {
+        receiveCompleteMessage(global.serverSocket, 1, global.deserializeBuffer);
+        vspeed = read_ubyte(global.deserializeBuffer);
+    }
+    else
```

Comment:
The else-branch runs on the following condition:
`built or global.updateType!=FULL_UPDATE)`
See the problem now?